src/app/pages/storage/modules/pool-manager/store/pool-manager.store.ts

```diff
@@ -117,6 +137,143 @@ export class PoolManagerStore extends ComponentStore<PoolManagerState> {
     }),
   );
 
+  disknumErrorMessage = helptext.manager_disknumErrorMessage;
+  disknumErrorConfirmMessage = helptext.manager_disknumErrorConfirmMessage;
+  disknumExtendConfirmMessage = helptext.manager_disknumExtendConfirmMessage;
+  vdevtypeErrorMessage = helptext.manager_vdevtypeErrorMessage;
+  exportedPoolsWarning = helptext.manager_exportedDisksWarning;
+
+  readonly poolCreationErrors$ = combineLatest([
```

Comment:
I think it would be easier to test if warnings related to logic is extracted into a separate service that uses PoolManagerStore.